<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroStock</class>
 <widget class="QMainWindow" name="RegistroStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Actualizar Stock</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Producto :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="select_Producto"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Cantidad :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_Cantidad"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ActualizarStock">
           <property name="text">
            <string>Actualizar Stock</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Gestionar Stock</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_NuevoProducto">
           <property name="text">
            <string>Nuevo Producto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Generar PDF Stock</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Imprimir Stock</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table_Stock">
      <column>
       <property name="text">
        <string>Producto</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cantidad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha Ultima Incorporacion</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>select_Producto</tabstop>
  <tabstop>input_Cantidad</tabstop>
  <tabstop>btn_ActualizarStock</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>table_Stock</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
